ddtrace/contrib/structlog/__init__.py

```diff
@@ -0,0 +1,55 @@
+"""
+Datadog APM traces can be integrated with the logs produced by structlog by:
+
+1. Having ``ddtrace`` patch the ``structlog`` module. This will add a
+processor in the beginning of the chain that adds trace attributes
+to the event_dict
+
+2. Ensure the configuration has at the minimum a processor chain that outputs
+to a log location without errors.
+
+3. For log correlation between APM and logs, the easiest format is via JSON
+so that no further configuration needs to be done
```

Comment:
```suggestion
so that no further configuration needs to be done in the datadog agent. 
```

Reply:
It's not really done in the agent, but rather in the backend via a logs pipeline